server/index.js

```diff
@@ -19,6 +19,12 @@ app.get('/api/health-check', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products', (req, res, next) => {
+  db.query('select * from "products"')
```

Comment:
We don't want the `longDescription` for this query.

Reply:
Okay!